### DISCUSSION TITLE
Tardis Coords Traveling

### DISCUSSION THREAD

**Discussion by mehboss** [eccentricdevotion/TARDIS#667] [General]
Hello. When I attempt to travel to specific coords using the Tardis plugin, it relays a message that says "An internal error occured while attempting to perform this command". I have tested it on a test server with just the tardis plugin and it works perfectly fine. I was wondering if anyone has any idea as to why this is happening. I have completely reset the tardis configuration, files, and world.

03.10 19:13:46 [Server] INFO [TARDIS] You are running TARDIS version: 3.8.2-b1666 with CraftBukkit git-Spigot-79a30d7-acbc348 (MC: 1.12.2)
Null error in console:
https://pastebin.com/2SjQQupV

**Comment by eccentricdevotion** [comment]
Pretty sure this has been fixed in later versions of the plugin...
